java-checks/src/main/java/org/sonar/java/checks/AbsOnNegativeCheck.java

```diff
@@ -39,6 +39,7 @@
 import java.util.List;
 
 @Rule(key = "S2676")
+// TODO
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&pullRequest=1950&issues=AWJJblOIuc1UcxX5-SQo&open=AWJJblOIuc1UcxX5-SQo)

[ISSUE_KEY]: AWJJblOIuc1UcxX5-SQo